sdk/tables/Azure.Data.Tables/samples/Sample4QueryEntities.md

```diff
@@ -0,0 +1,77 @@
+# Query Entities
+This sample demonstrates how to query a table for entities. To get started, you'll need access to either a Storage or Cosmos DB account.
+
+## Create a `TableClient`
+A `TableClient` is needed to perform table-level operations like inserting and deleting entities within the table. There are two ways to get a `TableClient`:
+- Call `GetTableClient` from the `TableServiceClient` with the table name.
+- Create a `TableClient` with a SAS URI, an endpoint and `TableSharedKeyCredential`, or a connection string.
+
+If the table has not been created in the service, call `Create`.
+
+```C# Snippet:TablesSample2CreateTableWithTableClient
+// Construct a new <see cref="TableClient" /> using a <see cref="TableSharedKeyCredential" />.
+var client = new TableClient(
+    tableName,
+    new Uri(storageUri),
+    new TableSharedKeyCredential(accountName, storageAccountKey));
+
+// Create the table in the service.
+client.Create();
+```
+
+## Query entities with `filter`
+To query entities that satisfy a specified filter, call `Query`, specify the desired entity return type, and pass in a filter in the form of either an OData formatted string or a LINQ expression.
+
+### OData formatted string
+Here is a query returning a collection of `TableEntity` objects that possess the specified partition key.
+
+```C# Snippet:TablesSample4QueryEntitiesFilter
+Pageable<TableEntity> queryResultsFilter = client.Query<TableEntity>(filter: $"PartitionKey eq '{partitionKey}'");
+
+// Iterate the <see cref="Pageable"> to access all queried entities.
+foreach (TableEntity qEntity in queryResultsFilter)
+{
+    Console.WriteLine($"{qEntity.GetString("Product")}: {qEntity.GetDouble("Price")}");
+}
+
+Console.WriteLine($"The query returned {queryResultsFilter.Count()} entities.");
+```
+
+### LINQ expression
+Here is a query returning a collection of the strongly-typed `OfficeSupplyEntity` objects that cost at least $6.00.
```

Comment:
for people that skipped the `Create and Delete Entities` samples, we should put a link here to so they know what the `OfficeSupplyEntity` is